src/components/MobileFooter/MobileFooter.module.scss

```diff
@@ -0,0 +1,48 @@
+@import "../../assets/styles/theme";
+
+.containerGridDark {
+  display: grid;
+  grid-template-columns: 52% auto;
+  grid-template-rows: 45% 45% 10%;
+  padding: 6%;
+}
+
+.containerGridDark > div {
+  text-align: left;
+  color: $white;
+}
+
+.containerGridLight > div {
+  text-align: left;
+  color: $black;
+}
+
+.mainContainerDark {
+  background: linear-gradient(180deg, $dark-footer-top-mobile 15%, $dark-footer-bottom-mobile 100%);
+}
+
+.mainContainerLight {
+  background: white;
+}
+
+.containerLinks {
+  display: flex;
+  a * {
+    padding: 10%;
+    margin-bottom: 20%;
+  }
+}
+
+h2 {
+  font-weight: bold;
+  font-size: 90%;
```

Comment:
Używanie % w fontach to raczej zła praktyka, używajmy normalnie px tak jak jest w Figmie. Bo z tego co widziałem jak sobie odpaliłem stronkę to niektóre teksty mają inne rozmiary czcionki niż jest to na Figmie.
Chyba że jest to czymś uzasadnione Mateusz?
A wy co myślicie chłopaki @kamil-ciesla @TheFenix2000

Reply:
jestem za px, a najlepsze by były remy wszędzie

Reply:
Ja jestem za remami, najmniejsza szansa że coś wykipi między mobile a desktop

Reply:
> Ja jestem za remami, najmniejsza szansa że coś wykipi między mobile a desktop

@szymonqqaz moze by tak to gdzieś uwspólnić i dodać taska `Correct font-sizes everywhere` w sensie żeby dać remy i no np tytuł taki był tyle i tyle, a elementy np h2 zawsze miały tyle i tyle, na konie cprojektu oczywiście
Co sądzicie